osu.Game/Screens/Ranking/ResultsScreen.cs

```diff
@@ -330,6 +330,9 @@ private void onStatisticsStateChanged(ValueChangedEvent<Visibility> state)
 
         public bool OnPressed(KeyBindingPressEvent<GlobalAction> e)
         {
+            if (e.Repeat)
```

Comment:
I think this should repeat, though the transition is abrupt right now.
![Screen Recording 2021-11-17 at 9 37 37 PM](https://user-images.githubusercontent.com/35318437/142358314-3e15a364-de3b-41eb-bb54-b984aa814921.gif)

Reply:
why?

Reply:
I slightly agree with the toolbar/overlay toggling to repeat and this is a similar case. Sure, fixing the animation is out of scope here, but if I can make it look less broken in a follow up PR, should it be a consideration?

Reply:
I think i'd sooner make this a non-toggle. Select actions are generally supposed to be one directional. This is an outlier.